src/status_im2/contexts/communities/actions/community_rules/view.cljs

```diff
@@ -0,0 +1,16 @@
+(ns status-im2.contexts.communities.actions.community-rules.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im2.contexts.communities.actions.community-rules.style :as style]
+    [utils.re-frame :as rf]))
+
+(defn view
+  [id]
+  (let
+    [rules (rf/sub [:communities/rules id])]
+    [rn/view {:style style/community-rule}
+     [quo/text
+      {:style  style/community-rule-text
+       :weight :regular
+       :size   :paragraph-2} rules]]))
```

Comment:
This style rule is important in lisps https://guide.clojure.style/#vertically-align-fn-args